spec/models/user_valid_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.describe User do
+  describe '#presenseメソッド' do
+    context 'emailが存在する場合' do
+      it 'valid?がtrueになる' do
+        user = User.new(email: 'hoge@example.com', password: 'password')
+        expect(user.valid?).to eq true
+      end
+    end
```

Comment:
`ActiveRecord::Base`を継承したクラスを用いてRspecを書く場合は、FactoryBotを使おう 👀 
また、letを使おう。

```suggestion
    context 'emailが存在する場合' do
      let(:user) { build(:user, email: 'hoge@example.com') }

      it 'valid?がtrueになる' do
        expect(user.valid?).to eq true
      end
    end
```